### ISSUE TITLE
About the five-degradation table in paper

### ISSUE THREAD

**Issue by hagerwang** [c-yn/AdaIR#10]
In the top super-row of the five-task table mentioned in the paper, was the accuracy of the single-task models trained with your training settings? I did not find the accuracies provided in your paper in these papers.

**Comment by hagerwang** [comment]
got it, thanks a lot!

Yuning Cui ***@***.***> 于2024年7月24日周三 15:09写道：
